### ISSUE TITLE
Failed to get caps when using daA3840-45uc in Jetson Orin

### ISSUE THREAD

**Issue by juhaautioniemi** [basler/gst-plugin-pylon#27]
I have multiple cameras and I am trying to publish stream from cameras to RTSP server.  Most of them works fine, but daA3840-45uc camera seems to fail when using Jetson Orin Developer Kit. In Windows 10 all cameras work. All cameras also work fine in both Jetson and Windows when using another pipeline. What could be the problem?

Here is the GStreamer short log, longer debug log is attached:

[debug_log.txt](https://github.com/basler/gst-plugin-pylon/files/10312701/debug_log.txt)

```
Setting pipeline to PAUSED ...
Opening in BLOCKING MODE
Pipeline is live and does not need PREROLL ...
Progress: (open) Opening Stream
Progress: (connect) Connecting to rtsp://localhost:8554/40257292
Progress: (open) Retrieving server options
Progress: (open) Opened Stream
Setting pipeline to PLAYING ...
New clock: GstSystemClock
Progress: (request) Sending RECORD request
Redistribute latency...
NvMMLiteOpen : Block : BlockType = 4
===== NVMEDIA: NVENC =====
NvMMLiteBlockCreate : Block : BlockType = 4
H264: Profile = 66, Level = 0
NVMEDIA: Need to set EMC bandwidth : 141000
NVMEDIA_ENC: bBlitMode is set to TRUE
ERROR: from element /GstPipeline:pipeline0/GstPylonSrc:pylonsrc0: Failed to get caps.
Additional debug info:
../ext/pylon/gstpylonsrc.c(467): gst_pylon_src_get_caps (): /GstPipeline:pipeline0/GstPylonSrc:pylonsrc0:
Node is not writable.
Execution ended after 0:00:00.759384283
Setting pipeline to NULL ...
```

GStreamer pipelines that I have used for testing in Jetson Orin:

daA3840-45uc  => NOT WORKING
```
gst-launch-1.0 pylonsrc capture-error=skip device-serial-number="40198132" ! "video/x-raw,width=1920,height=1080,framerate=5/1,format=YUY2" ! nvvidconv ! nvv4l2h264enc ! queue ! rtspclientsink location = rtsp://localhost:8554/40198132
```

daA1920-160uc => OK
```
gst-launch-1.0 pylonsrc capture-error=skip device-serial-number="40165426" ! "video/x-raw,width=1920,height=1080,framerate=5/1,format=YUY2" ! nvvidconv ! nvv4l2h264enc ! queue ! rtspclientsink location = rtsp://localhost:8554/40165426
```

Another daA3840-45uc => NOT WORKING
```
gst-launch-1.0 pylonsrc capture-error=skip device-serial-number="40257292" ! "video/x-raw,width=1920,height=1080,framerate=5/1,format=YUY2" ! nvvidconv ! nvv4l2h264enc ! queue ! rtspclientsink location = rtsp://localhost:8554/40257292
```

a2A1920-51gcBAS => OK
```
gst-launch-1.0 pylonsrc capture-error=skip device-serial-number="40204286" ! "video/x-raw,width=1920,height=1080,framerate=5/1,format=YUY2" ! nvvidconv ! nvv4l2h264enc ! queue ! rtspclientsink location = rtsp://localhost:8554/40204286
```

acA2500-14gc => OK
```

gst-launch-1.0 pylonsrc capture-error=skip device-serial-number="23751808" ! "video/x-raw,width=1920,height=1080,framerate=5/1,format=YUY2" ! nvvidconv ! nvv4l2h264enc ! queue ! rtspclientsink location = rtsp://localhost:8554/23751808
```

Pipeline that works in Windows 10 with all cameras:
```
gst-launch-1.0 pylonsrc capture-error=skip device-serial-number="40257292" ! "video/x-raw,width=1920,height=1080,framerate=5/1,format=YUY2" ! videoconvert ! x264enc ! queue ! rtspclientsink location = rtsp://localhost:8554/40257292
```

Pipeline that works in all cameras in Jetson Orin 
```
gst-launch-1.0 pylonsrc capture-error=skip device-serial-number="40198132" ! "video/x-raw,width=1920,height=1080,framerate=5/1,format=YUY2" ! nvvidconv ! queue ! nvjpegenc ! queue ! tcpclientsink host=localhost port=50006
```

**Comment by thiesmoeller** [comment]
Hi @juhaautioniemi, 
we made a hotfix for 0.5.0 -> v0.5.1 which fixes issue #26 -> with a similar failure state.
As I can't currently test with your specific device, would be great if you give the latest 0.5.1 release a try.

**Comment by juhaautioniemi** [comment]
Hi and thanks. Release 0.5.1 solves this issue.
